ios/Capacitor/Capacitor/CapacitorBridge.swift

```diff
@@ -209,6 +209,10 @@ internal class CapacitorBridge: NSObject, CAPBridgeProtocol {
         observers.append(NotificationCenter.default.addObserver(forName: type(of: self).tmpVCAppeared.name, object: .none, queue: .none) { [weak self] _ in
             self?.tmpWindow = nil
         })
+
+        #if DEBUG
+        self.webView?.setInspectableIfRequired(true)
```

Comment:
A super minor nit/question: Are we planning to make this a configuration option or have cases where this would be set to false later? Only having the one call site and passing a `true` literal makes me wonder if it should just be in the method itself instead of passed as an argument.